src/traits/mod.rs

```diff
@@ -59,6 +62,7 @@ pub mod misc;
 pub mod button;
 pub mod label;
 pub mod _box;
+pub mod flow_box;
```

Comment:
This should be preceded by `#[cfg(feature = "gtk_3_12")]`.
And I don't see a similar import in `widgets/mod.rs` at all.